src/Components/test/E2ETest/Tests/RoutingTest.cs

```diff
@@ -808,7 +805,7 @@ public void CanArriveAtQueryStringPageWithDateTimeQuery()
             var dateTime = new DateTime(2000, 1, 2, 3, 4, 5, 6);
             var dateOnly = new DateOnly(2000, 1, 2);
             var timeOnly = new TimeOnly(3, 4, 5, 6);
-            SetUrlViaPushState($"/WithQueryParameters/Abc?&NullableDateTimeValue={dateTime.ToString("u")}&NullableDateOnlyValue={dateOnly.ToString("u")}&NullableTimeOnlyValue={timeOnly.ToString("u")}");
+            SetUrlViaPushState($"/WithQueryParameters/Abc?NullableDateTimeValue=2000-01-02%2003:04:05&NullableDateOnlyValue=2000-01-02&NullableTimeOnlyValue=03:04:05");
```

Comment:
Constructing URL via interpolation was causing issues (due to localization / date cultures). Testing with a hard-coded string for now.